coverage-tests/configuration/index.js

```diff
@@ -1,12 +1,13 @@
+let ref = '6f22b01f'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=9637824)
```suggestion
let ref = "6f22b01f"
```